lib/workers/branch/schedule.ts

```diff
@@ -67,6 +82,42 @@ export function hasValidSchedule(
   return [true, ''];
 }
 
+function cronMatches(cron: string, now: DateTime): boolean {
+  const parsedCron = parseCron(cron);
+
+  if (
+    parsedCron.hours.length !== 0 &&
```

Comment:
So the array will be empty when `*` is used?

I don't think those arrays will be ever empty.

https://www.npmjs.com/package/@cheap-glitch/mi-cron#parsecronexp-string-cronschedule

Reply:
@viceice thank you, I've checked and when we use *, the array will contain 60 items. Those array seem to be always non-empty.